ydmyterko/HM7/HW3.py

```diff
@@ -0,0 +1,9 @@
+def Number_of_characters(str) -> None:
```

Comment:
look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result

print(count_symbols_in_row("abracadabra"))